obstore/python/obstore/fsspec.py

```diff
@@ -542,5 +737,6 @@ def _register(protocol: str, *, asynchronous: bool) -> None:
                 "asynchronous": asynchronous,
             },  # Assign protocol dynamically
         ),
-        clobber=False,
+        # Override any existing implementations of the same protocol
+        clobber=True,
```

Comment:
Note, I changed this, because I think we want to override any of the default fsspec protocols